process/block/shardblock_test.go

```diff
@@ -4588,13 +4551,52 @@ func TestShardProcessor_restoreMetaBlockIntoPoolShouldPass(t *testing.T) {
 	metablockHashes := make([][]byte, 0)
 	metablockHashes = append(metablockHashes, metaHash)
 
+	metaBlockRestored, ok := poolFake.MetaBlocks().Get(metaHash)
+
+	assert.Equal(t, nil, metaBlockRestored)
+	assert.Equal(t, false, ok)
```

Comment:
you can use assert.False instead of assert.Equal here.

Reply:
done